### ISSUE TITLE
Understanding s2v_mc

### ISSUE THREAD

**Issue by pandalgo** [Hanjun-Dai/graph_comb_opt#21]
Hi, 
Thank you for sharing your code.

I am trying to understand how to write the equation for Q in s2v_mvc, given the additional auxiliary input. Following the notation in the paper, is it like this :  
<img width="271" alt="q_function" src="https://user-images.githubusercontent.com/63570035/79087219-96b8e280-7d0c-11ea-8848-bcd0abf6c3a5.png">

Where _a^{T}_ is the auxiliary input and _θ_{8}_ is 3-dimensional. Am I right? If I am not, can you explain how to write the equation for s2v_mvc? 

Thank you!

**Comment by Hanjun-Dai** [comment]
Yes a^T is the auxiliary input which can be node degree, coverage information, etc. 
There are some differences:

1) I guess in my implementation I simply concat a^T with the relu part (i.e., remove \theta_8)
2) the summation is over entire graph, instead of the neighborhood.
